cmd/collector/app/tchannel_handler.go

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2019 The Jaeger Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package app
+
+import (
+	"github.com/uber/tchannel-go/thrift"
+
+	"github.com/jaegertracing/jaeger/thrift-gen/jaeger"
+	"github.com/jaegertracing/jaeger/thrift-gen/zipkincore"
+)
+
+type tchannelHandler struct {
```

Comment:
Maybe we can add an interface for tchannel handler, that way it's more clear this handler is satisfying the thrift definition?

``` 
type tchanHandler struct {
	zipkinSpansHandler ZipkinSpansHandler
	jaegerSpansHandler JaegerBatchesHandler
}
```

Reply:
it already implements existing interfaces, no benefit of defining more. I added comments and the API checks to the test file,